src/syscheckd/fim_db.c

```diff
@@ -1150,3 +1152,27 @@ void fim_db_callback_sync_path_range(__attribute__((unused))fdb_t *fim_sql, fim_
     free_entry(entry);
     os_free(plain);
 }
+
+int fim_db_get_count_entry_data(fdb_t * fim_sql) {
+    int res = sqlite3_step(fim_sql->stmt[FIMDB_STMT_GET_COUNT_DATA]);
```

Comment:
Add `fim_db_clean_stmt(fim_sql, FIMDB_STMT_GET_COUNT_DATA)` before `sqlite3_step()`. The same goes for `fim_db_get_count_entry_path()`.